docs/Governance.md

```diff
@@ -0,0 +1,73 @@
+
+New core contributors can be nominated by any existing core contributors. Once they have been nominated, there will be a vote in the [private OpenML core email list](https://lists.lrz.de/mailman/listinfo/openml-core) by the current core contributors. While it is expected that most votes will be unanimous, a two-thirds majority of the cast votes is enough. The vote needs to be open for at least 1 week.
+
+Core contributors that have not contributed to the project (commits or GitHub comments) in the past 12 months will become emeritus core contributors and recant their commit and voting rights until they become active again. The list of core contributors, active and emeritus (with dates at which they became active) is public on the OpenML website.
+
+
+#### Steering Committee
+
+The Steering Committee (SC) members are core contributors who have additional responsibilities to ensure the smooth running of the project. SC members are expected to participate in strategic planning, join monthly meetings, and approve changes to the governance model. The purpose of the SC is to ensure a smooth progress from the big-picture perspective. Indeed, changes that impact the full project require a synthetic analysis and a consensus that is both explicit and informed. In cases that the core contributor community (which includes the SC members) fails to reach such a consensus in the required time frame, the SC is the entity to resolve the issue. 
+
+The Steering Committee consists of community representatives and partner representatives.
+
+Community representatives of the SC are nominated by a core contributor. A nomination will result in a discussion that cannot take more than a month and then a vote by the core contributors which will stay open for a week. Partner institutions who enter a collaboration agreement with the OpenML Foundation can nominate a representative on the Steering Committee, if so agreed in the collaboration agreement.
+
+SC membership votes are subject to a two-third majority of all cast votes as well as a simple majority approval of all the current SC members. When decisions are escalated to the steering committee (see the decision making process below), and no consensus can be found within a month, the SC can decide with a simple majority vote. In addition, to ensure the community character of the OpenML project, community representatives always carry at least 51% of the vote.
```

Comment:
These aspects have changed a bit. Maybe come back to this when the final version is approved?

Reply:
Yes, agreed

Reply:
I set the PR to WIP to show that we wait for this. Is that ok?

Reply:
ok